backend/modules/sse/tests/test_sse_consumer_impl.py

```
@@ -3,6 +3,7 @@
 
 import pytest
 
+from modules.prices.schemas import Client
```

Pylint (3.3.7)

[error] Cannot import 'modules.prices.schemas' due to 'invalid syntax (modules.prices.schemas, line 17)'

(E0001)